dingdongg/spotify-payment
# M1 restructure

**Description**

- restructured `views` folder by grouping components into admin and member directories
- cleaned up code (whitespace, unused imports)

**To Test**

- Just `npm run dev` and make sure that all the pages and their contents load correctly
